BinaryEncoding.md

```diff
@@ -119,11 +119,15 @@ The type section declares all function signatures that will be used in the modul
 | entries | `type_entry*` | repeated type entries as described below |
 
 #### Type entry
-| Field | Type | Description |
+| Field | Type/Value | Description |
 | ----- |  ----- | ----- |
+| constructor | `0x05` | the function type constructor |
```

Comment:
I see, so if the was greater-than a fixed base, you'd subtract the base and use that as the index into the types section?  (Or we could do the analogous thing with signed `varint32` and negatives.)